JournalCoreData/JournalCoreData/Entry+Convenience.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Entry+Convenience.swift
+//  JournalCoreData
+//
+//  Created by Angel Buenrostro on 2/11/19.
+//  Copyright © 2019 Angel Buenrostro. All rights reserved.
+//
+
+import CoreData
+
+extension Entry {
+    
+    convenience init(title: String, bodyText: String, timestamp: Date = Date(), identifier: String = "entry", context: NSManagedObjectContext = CoreDataStack.shared.mainContext) {
```

Comment:
Instead of same default value give them different unique values. So we can compare them. Which we will compare them by their IDs in the next assignments
```suggestion
    convenience init(title: String, bodyText: String, timestamp: Date = Date(), identifier: String = UUID().uuidString, context: NSManagedObjectContext = CoreDataStack.shared.mainContext) {
```